catalog/video/video-admin-plugin/src/main/java/org/codice/alliance/video/ui/service/StreamMonitorHelper.java

```diff
@@ -133,18 +160,46 @@ public void callStopMonitoringStreamByServicePid(String servicePid) {
                 .collect(Collectors.toList());
     }
 
-    public void init() {
-        registerMbean();
+    private String commaSeparatedListOfIPv4(NetworkInterface networkInterface) {
+        return Collections.list(networkInterface.getInetAddresses())
+                .stream()
+                .filter(IPV4_FILTER)
+                .map(InetAddress::getHostAddress)
+                .collect(Collectors.joining(","));
     }
 
-    public void setContext(BundleContext context) {
-        this.context = context;
+    private String formatNeworkInterface(NetworkInterface networkInterface) {
+        return networkInterface.getDisplayName() + " (" + commaSeparatedListOfIPv4(networkInterface)
+                + ")";
+    }
+
+    @Override
+    public Map<String, String> networkInterfaces() {
```

Comment:
Also, I'd be curious to get @andrewkfiedler's take on this. Should this class return a `Map` of pre-formatted values, or return a JSON object in a string and let the UI do the formatting that `formatNeworkInterface()` does?